int32 pid_axis
float64 kp
float64 ki
float64 kd
float64 i_max
float64 i_min
float64 vz_min
float64 vz_max
---
bool success